doc/docs.md

```diff
@@ -1025,6 +1030,8 @@ posts_repo := new_repo<Post>(db)
 user := users_repo.find_by_id(1)?
 post := posts_repo.find_by_id(1)?
 ```
+Generics enables you to write flexible, reusable functions and types that can work with any type.
```

Comment:
"enables" should be "enable".  "types that can work with any type." is awkward/redundant phrasing.  Perhaps something like "placeholders for any type."